### ISSUE TITLE
Check ICA for amplitude issue

### ISSUE THREAD

**Issue by ElineOppersma** [resurfemg-org/ReSurfEMG#123]
ICA makes a decomposition in the components and scaling factors, but afterwards the components should be multiplied by the scaling factor. Check difference in amplitude values with gating.

**Comment by ElineOppersma** [comment]
Improve ICA to Picard function to clearify the amplitude issue. Goal for next release is user chooses between valid Picard leads or other expert recommended algorithm.

**Comment by drcandacemakedamoore** [comment]
PR #153

**Comment by wbaccinelli** [comment]
Blocked by: #155

**Comment by drcandacemakedamoore** [comment]
No longer blocked (155 awaits review). We should not wait for a cohere rewrite of Rob's next improvement, as we want a release before Biosignals. We can add other algorithms, but implementation of Picard as standard requires environment changes, therefore should be discussed.
We should open a new issue for Rob's version to come out of a new notebook if  a new notebook isn't written by this close.

**Comment by drcandacemakedamoore** [comment]
#188

**Comment by ElineOppersma** [comment]
Amplitude changes with every preprocessing step. #155 makes the ICA output more similar to the original values of the EMG (same order of magnitude).
